pallets/validators-manager/src/lib.rs

```diff
@@ -619,6 +630,58 @@ impl<T: Config> Pallet<T> {
         Ok(hex::encode(EthAbiHelper::generate_eth_abi_encoding_for_params_only(&eth_description)))
     }
 
+    /// This function generates the data needed to generate a confirmation for registering a new
+    /// collator. The implementation must match this schema:
+    /// https://github.com/Aventus-Network-Services/avn-bridge/blob/v1.1.0/contracts/AVNBridge.sol#L344-L345
+    pub fn convert_collator_registration_action_to_confirmation_signing_data(
+        action_id: &ActionId<T::AccountId>,
+        enable_data_logs: bool,
```

Comment:
I am not sure about this. I think we should have a better way to handle enabling/disabling logs.

Reply:
I think we should remove the flag and always have the logs for now at the very least